cloud/cluster/local/config.go

```diff
@@ -0,0 +1,15 @@
+package local
+
+import (
+	"github.com/scootdev/scoot/cloud/cluster"
+	"github.com/scootdev/scoot/cloud/cluster/memory"
+)
+
+type ClusterLocalConfig struct {
+	RegexCapturePort string
```

Comment:
Document this parameter.

(Also, why is it configurable? Do we need configurability right? If not, I'd just hardcode it saying "yes, this is ugly, but it works for now)